#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_openser.cfg.dpatch by  <jblache@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Remove C-style commented fork-no, otherwise the default config
## DP: file will trigger the test for fork=no in the initscript.

@DPATCH@
diff -urNad openser-1.1.0~/etc/openser.cfg openser-1.1.0/etc/openser.cfg
--- openser-1.1.0~/etc/openser.cfg	2006-02-15 19:23:46.000000000 +0100
+++ openser-1.1.0/etc/openser.cfg	2006-09-08 14:37:46.165753127 +0200
@@ -7,13 +7,8 @@
 # ----------- global configuration parameters ------------------------
 
 debug=3            # debug level (cmd line: -dddddddddd)
-fork=yes
-log_stderror=no    # (cmd line: -E)
-
-/* Uncomment these lines to enter debugging mode 
-fork=no
-log_stderror=yes
-*/
+fork=yes           # Set to no to enter debugging mode
+log_stderror=no    # (cmd line: -E) Set to yes to enter debugging mode
 
 check_via=no	# (cmd. line: -v)
 dns=no          # (cmd. line: -r)
